dune/gdt/spaces/rt/interface.hh

```diff
@@ -179,6 +179,11 @@ public:
   {
     DUNE_THROW(NotImplemented, "RT spaces do not implement constraints!");
   }
+
+  static constexpr bool associates_data_with(int codim)
+  {
+    return codim < dimDomain;
```

Comment:
This is actually (see Brezzi, Fortin, 1991, p. 117):
* order 0: codim 1
* orders >= 1: codim 0 and 1